### PR TITLE
fix: navigateUrl to accept normal string as url

### PR DESCRIPTION
**Pull request by sagardspeed2**
## Describe your changes
accept normal string as url in navigateUrl

### Screenshots [Optional]
before: 
image

yaml
```yaml
Button:
    label: navigate url
    onTap:
      navigateUrl: /forms
```

## Issue ticket number and link

Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] I have added tests
- [x] I have added a changeset `pnpm changeset add`
- [x] I have added example usage in the kitchen sink app

### CODE REVIEW

File: packages/framework/.pnpm-debug.log [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (anserwaseem) [inline packages/framework/.pnpm-debug.log:1]: is this file supposed to be here?

Comment (sagardspeed2) [reply packages/framework/.pnpm-debug.log:1]: sorry, let me remove that

**Review by evshi — COMMENTED** [review]
There's a lot of unnecessary changes here can we please clean this up?

File: packages/runtime/src/runtime/hooks/useNavigateUrl.ts [hunk 2]
```diff
@@ -55,20 +66,20 @@ export const useNavigateUrl: EnsembleActionHook<NavigateUrlAction> = (
...
         ? undefined
-        : evaluateInputs(action.inputs, screenModal, context);
+        : evaluateInputs(action.inputs, screenModel, context);
 
-      navigateUrl(evaluatedUrl, navigate, evaluatedInputs);
+      navigateUrl(resolvedUrl, navigate, resolvedInputs);
     },
     { navigate },
-    [action, screenModal],
+    [action],
```
Comment (evshi) [inline packages/runtime/src/runtime/hooks/useNavigateUrl.ts:82]: is model not needed here?

**Review by evshi — APPROVED** [review]

**Pull request merged**